db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -492,6 +492,11 @@ class NonBatchedOpsStressTest : public StressTest {
     ReadOptions read_opts_copy = read_opts;
     std::string read_ts_str;
     Slice read_ts_slice;
+    if (FLAGS_user_timestamp_size > 0) {
```

Comment:
Thanks for the fix @jowlyzhang - makes sense!

One thing I'm wondering is if we should update this check (on line 522 with the patch):

```
if (!FLAGS_skip_verifydb && !read_opts_copy.timestamp &&
```

so that it checks `!read_older_ts` instead of `!read_opts_copy.timestamp` ?

Reply:
Good catch! I agree with you, checking `ReadOptions.timestamp` is set doesn't help much. I guess this branch wasn't hit as much because delete and delete range percentages are usually low compared to write percentage.